src/amazonqGumby/activation.ts

```diff
@@ -48,4 +56,27 @@ export async function activate(context: ExtContext) {
             vscode.commands.executeCommand('markdown.showPreview', vscode.Uri.file(transformByQState.getPlanFilePath()))
         })
     )
+
+    // Register an activation event listener to determine when the IDE opens, closes or users
```

Comment:
Just making sure, have you validated that you're able to receive this metric? IIRC VS Code doesn't guarantee to let extension processes wrap up when closing the IDE, so there's a chance this may not go through reliably.

Reply:
It may be recorded in telemetry but are not emitted at that very moment. so they are accumated in cache when IDE closes and emitted only when user reopens and metrics are still in cache